deployments/ansible/molecule/custom_vars/custom_collector_config.yml

```diff
@@ -24,9 +24,9 @@ processors:
 
 exporters:
   logging/debug:
-    loglevel: debug
+    verbosity: detailed
   logging/info:
-    loglevel: info
+    verbosity: normal
```

Comment:
@atoulme We should revert this since this ansible test is specifically testing with an older version of the collector.